lessons/Data-Science/box-plots/langs/en-us/index.adoc

```diff
@@ -33,93 +32,54 @@
 
 | Preparation
 |
-@preparation{
-- There is an optional kinesthetic activity in this lesson that requires a ball of play-dough for each group of 3.
-- There are Desmos formative assessments at the end of each lesson section.
-  * Open the assessment you plan to administer (@assessment{box-plots-check1-desmos}, @assessment{box-plots-check2-desmos}, etc) and make a code or link to share with students.
-  * If you are using our Google Slides, add the appropriate link to the slide deck.
-  * _If you're a first-time Desmos user, fear not!_ @dist-link{pages/desmos-instructions.adoc, _Here's what you need to do._}
-}
 
-| Assessments
-| [.assessment-links]
-@assessments
 
 |===
 
-== How Typical is the Average?
+== The Five-Number Summary
 
 === Overview
-Students are introduced to the notion of _spread_ in a dataset. They learn about how to use range, quartiles, and interquartile range to talk about variation.
-
-=== Launch
-When we explored measures of center, _we determined that the average pet weighs almost 40 pounds_.
-
-Once we've calculated our summary value, it's important to ask: _How typical is the average?_ +
-We'd expect points to deviate from the mean in any sample!
 
-@QandA{
-@Q{Does the average tell us the "typical" weight of animals at the shelter?}
-@A{Maybe!
-- Perhaps all of the pets weigh between 35 and 45 pounds, with every pet close to the mean. 
-- But perhaps all of the pets are super small or huge, and none are anywhere _near_ the mean!
-}
-}
+Students learn about how to use range, quartiles, and interquartile range to talk about variability.
 
-@slidebreak
+=== Launch
 
-To learn about how evenly distributed the data is we can start by ordering the data from least to greatest. From there we can identify and compute a number of values that will give us more information.
+The three dot plots shown below are differently sized samples drawn from the *same dataset*. _Each of the three samples has a median of 60._
```

Comment:
@retabak I have read the words in lines 47 - 61 over and over and I think they are unnecessarily complicate. 
Why do you think it's important for students to know that these samples came from the same dataset? 

Even once I wrap my brain around it, I'm distracted by thinking about how unlikely it is that 3 random samples from a dataset would all have the same median...

Reply:
Totally happy to change this. I was just doing what the issue said to do. (Requested by the teacher who reviewed our stuff, at the request of ES.)

Curious to hear suggestions about how you think it ought to be set up.

Reply:
Cool. I'll propose an alternative.